pkgs/development/python-modules/pyqt/pyqt5-pyproject-darwin-min-sdk.patch

```diff
@@ -0,0 +1,8 @@
+diff --git a/pyproject.toml b/pyproject.toml
+--- a/pyproject.toml
++++ b/pyproject.toml
+@@ -16,4 +16,5 @@
+
+ [tool.sip.project]
+ verbose = true
```

Comment:
Managing inter-dependent patches may be cumbersome in the long-term.

Reply:
Do you know how to fix it? Sry I'm really bad with all python stuff, I'm more cpp and go guy(

Reply:
Before your change, this was done not in `patches` but in `postPatch`.

Reply:
Yes, I know, you think `postPatch` will be better for this case?

Reply:
Yes